docs/src/pages/components/app-bar/app-bar.md

```diff
@@ -43,6 +43,33 @@ A side searchbar.
 
 You can use the `useScrollTrigger()` hook to respond to user scroll actions.
 
+## Placement
+
+When using Appbar `variant="fixed"` you need to have extra space for the content to show below
+& not under. There are 2 ways to do it. Either use `theme.mixins.toolbar` like;
+
+```jsx
+const useStyles = makeStyles(theme => ({
+  offset: {
+    ...theme.mixins.toolbar,
+    flexGrow: 1
```

Comment:
Do we need flexGrow in this case?

Reply:
no `flexGrow` is not needed, I'll update the PR